create_cards/views.py

```diff
@@ -172,6 +172,25 @@ def get_effect_for_info(request):
     else:
         return JsonResponse({"error": "Unsupported request type"})
 
+def get_power_points(request):
+    """
+        A POST view to get a calculated power_points for some info set by a form in the UX.
+    """
+    if request.method == "POST":
+        info = json.load(request)
+        card_info = info["card_info"]
+        if "cost" in card_info:
+            card_info["cost"] = int(card_info["cost"])
+        if "strength" in card_info:
+            card_info["strength"] = int(card_info["strength"])
+        if "hit_points" in card_info:
+            card_info["hit_points"] = int(card_info["hit_points"])
+        Analytics.log_amplitude(request, "Create Card Get Power Points", {})
+        power_points = Card(card_info).power_points_value()
+        return JsonResponse({"power_points": power_points})
+    else:
+        return JsonResponse({"error": "Unsupported request type"})
```

Comment:
I haven't looked how everything is wired up yet, but usually "what http methods do I support" is handled by implementing get() vs post(), or annotations. You also just check if != post and return at the beginning of the method and then do the rest.

Reply:
good thought, i switched this views file to use request_POST decorator throughout